### PR TITLE
feat: Nft details transfers tab 2

### LINKED ISSUE #147: NFT details transfers tab
A new tab called "Transfers" should be visible in the NFT details page with the following paginated table:
| Hash | Time | Token ID | Recipient | Sender |
|--------|--------|--------|--------|--------|
| th_k67Jg5p...n7 | 1 day ago 651434 | 8 | ak_QVSUoGr...YW | ak_2tw26Rw...z8 |

Example call: https://testnet.aeternity.io/mdw/v2/aex141/transfers/ct_2tw26RwgNADrpuCnrQWKPBH87bPxuRbLR1KLccS9ZJTUMMj4z8

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #147

## Demo
`ct_2tw26RwgNADrpuCnrQWKPBH87bPxuRbLR1KLccS9ZJTUMMj4z8`
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add transfer nft tab

### CODE REVIEW

File: src/components/NftTransfersPanel.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+async function loadPrevNftTransfers() {
+  await fetchNftTransfers({ queryParameters: nftTransfers.value.prev })
+}
+
+async function loadNextNftTransfers() {
+  await fetchNftTransfers({ queryParameters: nftTransfers.value.next })
+}
+
+await fetchNftTransfers({
```
Comment (lukeromanowicz) [inline src/components/NftTransfersPanel.vue:43]: I believe this should be executed on client side only due to using a window-dependent limit

File: src/components/NftDetailsPanel.vue [hunk 2]
```diff
@@ -104,6 +104,7 @@
 </template>
 
 <script setup>
+
```
Comment (lukeromanowicz) [inline src/components/NftDetailsPanel.vue:107]: Most of our files don't use that additional newline after `<script setup>` Is there a reason for this sudden change?

**Review by lukeromanowicz — CHANGES_REQUESTED** [review]
Very nice work, as always. :rocket: Almost perfect

File: src/components/NftTransfersTable.vue [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          </hint-tooltip>
+        </th>
+        <th>
+          Token ID
+          <hint-tooltip>
+            {{ nftsHints.transferTokenId }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Recipient
```
Comment (michele-franchi) [inline src/components/NftTransfersTable.vue:24]: I would add the Sender column before Recipient, I think it makes more sense logically

File: src/components/NftTransfersTableCondensed.vue [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+            </app-tooltip>
+          </th>
+          <td class="contract-call-transactions-table-condensed__data">
+            {{ transfer.tokenId }}
+          </td>
+        </tr>
+        <tr class="contract-call-transactions-table-condensed__row">
+          <th class="contract-call-transactions-table-condensed__header">
+            <app-tooltip>
+              Recipient
```
Comment (michele-franchi) [inline src/components/NftTransfersTableCondensed.vue:58]: Same here, I would add Sender column before Recipient

File: src/utils/hints/nftHints.js [hunk 5]
```diff
@@ -7,4 +7,9 @@ export const nftsHints = {
   extensions: 'Extensions that are implemented for this NFT according to the creator of the smart contract.',
   tokenLimit: 'The maximum number of NFTs that can be minted.',
   templateLimit: 'The maximum number of NFTs that can be minted per template.',
+  transferHash: 'The hash of the transaction that transfers the NFT.',
```
Comment (michele-franchi) [inline src/utils/hints/nftHints.js:10]:
```suggestion
  transferHash: 'The hash of the transaction that transferred the NFT.',
```

I would keep the same verb tense of the token id hint.

**Review by michele-franchi — CHANGES_REQUESTED** [review]
It looks good but I would apply some minor changes, please have a look at my comments.

File: src/components/NftDetailsPanel.vue (hunk 2 above)
Comment (janmichek) [reply src/components/NftDetailsPanel.vue:107]: oops, fixed

File: src/components/NftTransfersTableCondensed.vue (hunk 4 above)
Comment (janmichek) [reply src/components/NftTransfersTableCondensed.vue]: Make sense, fixed

File: src/components/NftTransfersTable.vue (hunk 3 above)
Comment (janmichek) [reply src/components/NftTransfersTable.vue]: Make sense, fixed

File: src/components/NftTransfersTableCondensed.vue (hunk 4 above)
Comment (janmichek) [reply src/components/NftTransfersTableCondensed.vue]: Requested design change as well

**Review by michele-franchi — APPROVED** [review]
Good job!

One hint is not working, have a look at my comment.

File: src/components/NftTransfersTable.vue [hunk 6]
```diff
@@ -0,0 +1,92 @@
+<template>
+  <table class="nft-transfers-table">
+    <thead>
+      <tr>
+        <th>
+          Hash
+          <hint-tooltip>
+            {{ nftsHints.contractId }}
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/components/NftTransfersTable.vue]: oops, I missed this somehow. Thanks, fixed

**Review by lukeromanowicz — APPROVED** [review]
Looks perfect

**Pull request merged**